src/styles/main.scss

```diff
@@ -15,11 +15,12 @@ body {
   width: 75px;
   height: 75px;
   border-radius: 5px;
-  color: #776e65;
-  box-sizing: border-box;
   text-align: center;
   vertical-align: center;
+  color: #776e65;
+  box-sizing: border-box;
   user-select: none;
+  transition: all 0.3s ease;
```

Comment:
don't use it like that, because it's the same as transition-duration 0.3s
It's too hard for the browser
Just list all params that will be chenged